frontend/src/styles/colors.sass

```diff
@@ -0,0 +1,9 @@
+$button-back-color: #1C2FBA
+$topic-back-color: #F8D5E8
+$notification-back-color: #61D4F6
+$black-font-color: #0B080A
+$icon-color: #6C707A
+$for-inactive-color: #CBC9D0
+$for-inactive-back-color: #E0E2F2
+$components-back-color: #F6F5FA
+$white-color: #FFFFFF
```

Comment:
Can you name them in more generic way?
Like 
White 
Gray-100
Gray-200
...
Black

accent-color = pink
main-color=blue
...=light-blue

Reply:
okay